cit3.0-web/public/index.html

```diff
@@ -56,6 +56,7 @@
       <!-- Snowplow stops plowing -->
 
     <title>Community Information Tool</title>
+    <title>
```

Comment:
I'm seeing quite a few things that I thought were already merged into develop in this PR, including this issue again - maybe merging from develop will get rid of it?

Reply:
Merging didn't fix but I fixed separately and pushed this file again.